packages/sw-cache-expiration/src/lib/cache-expiration.js

```diff
@@ -193,17 +200,34 @@ class CacheExpiration {
    * Expires entries, both based on the the maximum age and the maximum number
    * of entries, depending on how this instance is configured.
    *
+   * A mutex is used to ensure that there is only one copy of this asynchronous
```

Comment:
The comments hopefully explain what's going on, but just to provide some more context for the review: running simultaneous copies of this method asynchronously *shouldn't* be a problem, but since they all are reading from the same IDB source of truth when deciding what to delete, it's an optimization to run one at a time.

But you need to account for new things being added to IDB that might not have been picked up by the in-progress invocation, so we want to schedule a single additional invocation following the completion of the current one, saving the most recent `now` value to use when recursively invoking it.